src/Publicizer/Publicizer.csproj

```diff
@@ -27,7 +27,7 @@
     <PackageTags>msbuild accesschecks public publicizer</PackageTags>
     <PackageIcon>icon.png</PackageIcon>
     <PackageReadmeFile>README.md</PackageReadmeFile>
-    <Version>2.0.1</Version>
+    <Version>2.1.0</Version>
```

Comment:
I appreciate you trying to do as much work as possible in this PR to make it ready for a release, but I think bumping the version should be done separate from the new feature. Please revert, and I'll bump it before release.